html/sales.html

```diff
@@ -1,13 +1,12 @@
 <!DOCTYPE html>
 <html>
 <head>
+  <meta charset="utf-8">
+  <link href="../css/backstyle.css" type="text/css" rel="stylesheet"></style>
```

Comment:
Does this link work? Currently, your HTML document is in the top directory, then you're saying <link> ".." <--- look in the directory above the html document "/css/backstyle.css" <--- then in the directory above the html document look for a folder named css and a file in that folder named backstyle.css. I'm not sure you it's going to be able to find it like this.